tests/end2endtest/es_schema.yml

```diff
@@ -0,0 +1,109 @@
+        pull any Docker image.
+        Format: ORG/IMAGE:M.N.U
+        Optional, default: null.
+    docker_port_mapping:
+      type: object
+      description: |
+        Port mapping for Docker image.
+        Optional, default: straight mapping (i.e. same port on host and in image).
+        Note that each server defines the single port that is to be used for
+        testing, so if you want to use both the HTTP and HTTPS ports, define
+        a server for each.
+      additionalProperties: false
+      required: [image, host]
+      properties:
+        image:
+          type: integer
+          description: |
+            Port number provided by Docker image.
+            Mandatory.
+        host:
+          type: integer
+          description: |
+            Port number used on host (local) system.
+            Mandatory.
+    url:
+      type: string
+      description: |
+        URL of the WBEM server; for details see url parameter of
+        pywbem.WBEMConnection().
+        Mandatory.
+    default_namespace:
+      type: string
+      description: |
+        Default CIM namespace to use in the created pywbem.WBEMConnection object.
+        Optional, default: null.
+    capabilities:
+      type: array
+      description: |
+        List of capabilities implemented by the server.
+        Optional, default: no capabilities.
+        The capability names are specific to pywbem end2end testing and are
+        checked by the end2end testcases. If a testcase requires a particular
+        capability and the server does not implement it, the testcase is skipped.
+      items:
+        type: string
+        enum:
+          - interop                # Has an Interop namespace
+          - namespaces             # Supports management of namespaces
+          - profiles               # General support for mgmt profiles
+          - snia-smis              # Support for SNIA SMI-S profile
+          - snia-server            # Support for SNIA Server profile
+
+vault_server_schema:
+  $schema: http://json-schema.org/draft-07/schema#
+  type: object
+  additionalProperties: false
+  properties:
+    user:
+      type: [string, "null"]
+      description: |
+        User for logging on to the WBEM server, or null for not specifying a user.
+        Optional, default: null.
+    password:
+      type: [string, "null"]
+      description: |
+        Password of that user, or null for not specifying a password.
+        Optional, default: null.
+    no_verification:
+      type: boolean
+      description: |
+        Boolean controlling whether the returned server certificate is to be verified.
+        Optional, default: true.
+    ca_certs:
+      type: [string, "null"]
+      description: |
+        Path name of CA certificate file or path name of directory containing CA
+        certificate files to be used for verifying the returned server certificate,
+        or null for using the pywbem default locations.
```

Comment:
change for to if

Reply:
This is the input for something to happen, that's why I used "for". Otherwise it would need to say something like "if the pywbem default locations are to be used", which is probably more ugly.
Not changed, for now.